implementations/elixir/ockam/ockam/lib/ockam/api/response.ex

```diff
@@ -36,7 +36,7 @@ defmodule Ockam.API.Response do
       end
 
     base =
-      Header.encode(%Header{
```

Comment:
Let's rename `Response.encode/1` -> `Response.encode!/1` for consistency. We can address mixed use of `:bare` and `CBOR` later.

Reply:
Good point, I've changed all three 👌🏼